src/main/java/com/users/controller/UserController.java

```diff
@@ -25,9 +27,15 @@ public ResponseEntity<UserDto> createUser(@Valid @RequestBody UserDto user) {
     }
 
     @GetMapping()
-    public ResponseEntity<List<UserDto>> findAll() {
+    public ResponseEntity<Page<UserDto>> findAll(UserCriteria criteria) {
```

Comment:
aici ai incercat si paginarea?

Reply:
l am adaptat ca nu era adaptat, dar nu merge pentru ca nu poate sa faca o conversie din String in StringFilter, dar o sa ma uit sa rezolv. Daca nu pun un criteriu gen email sau id merge doar cu pageNo si pageSize.

Reply:
pai vezi ca e diferenta intre UserCriteria si UserCriteriaEasy, daia am si facut 2.. asta este in genul celui pe care ai vrut tu sa l faci